spec/lib/emis/payment_service_v2_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'emis/payment_service'
+
+describe EMIS::PaymentServiceV2 do
+  describe 'get_pay_grade_history' do
+    let(:edipi) { '1007697216' }
+
+    context 'with a valid request' do
+      it 'calls the get_pay_grade_history endpoint with a proper emis message' do
+        VCR.use_cassette('emis/get_pay_grade_history/success') do
```

Comment:
when i write specs with VCR i usually add `{ match_requests_on: %i[method uri headers body] }`, otherwise VCR does not check if the request headers or body match what is in the cassette. we've already run into bugs with that before when we introduced a bug that made us send the wrong request body but all the vcr specs still passed since it wasn't checking the bodies. this makes writing specs more complicated though since you usually need to stub out a few things like uuids and current time to match what's in the cassette.